svo (setup, misc, empty, funnies, dor).xml

```diff
@@ -22,7 +22,7 @@ svo.modules_version = svo.modules_version or {}
 svo.modules_version["svo (setup, misc, empty, funnies, dor)"] = 1
 svo.loader.setup = function()
 
-svo.version = "54"
+svo.version = "55"
```

Comment:
Another one for me. This should be 54 still since latest version is 53 and 54 was reverted

Reply:
For transparency sake (and if people already installed 54) you should use 55 here.